### PR TITLE
CPDP-2199: Further returns guidance page

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              )._1
+            )
+          }
+
+          checkIsTechnicalErrorPage(performAction("abc"))
+        }
+
+      }
+
+      "display the page" when {
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:134]: it'd be good to unit test the different h1's for the different user types as well - only individual "self" users are being tested atm

Comment (AndyHWChung) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:134]: making sure the different journey statuses are covered would also be good (`StartingNewDraftReturn`, `FillingOurReturn` and `StartingAmendReturn`). It'd be enough to make sure the tests for each user type covers all the different journey statuses that are allowed. we don't necessarily need every user type with all journey status types

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:134]: Is it okay how we did for other controllers like `ReliefDetailsControllerSpec`?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceController.scala [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          _ => None,
+          _.newReturnTriageAnswers.fold(
+            _.representativeType(),
+            _.representativeType()
+          )
+        ),
+        _.draftReturn.representativeType()
+      )
+
+  private def withState(request: RequestWithSessionData[_])(
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceController.scala:90]: could you rename this to `withJourneyState` as that makes it more clear what state it is referring to

**Pull request merged**